### ISSUE TITLE
To be refined: use GUID of 61850-6-100 DRAFT

### ISSUE THREAD

**Issue by Sander3003** [com-pas/compas-architecture#119]
As a user
I want to use 61850 as standard as possible including the use of GUID
So I can exchange SCL files with other parties/software tools in the most interopable way

Background:
61850-6-100 DRAFT contains  a way to use a GUID

**Comment by dlabordus** [comment]
Looking at the draft, the 61850-6-100 has one disadvantage about how it's setup, it depend on the file 'scl.xsd', but we have 3 versions of the scl.xsd.
This would mean that even because there is only 1 61850-6-100 DRAFT XSD, we probably need to add this XSD to all 3 modules for the 3 different versions if we need Java Class being made from it.

**Comment by Sander3003** [comment]
The idea is to add this 6-100 xsd to the scl.xsd and generate java classes afterwards (so it includes both XSD files). 6-100 depends on the SCL.xsd file.

This could lead to combability issues in the future. 61850-6-100 version can change and scl.xsd can change in the future as well. This could lead to a lot of versions in the future with their challenges.

CoMPAS private extensions are fully decoupled of the scl.xsd file. This gives more flexibility.

@jeanetiennelemaire suggest WG10 that 6-100 should be independent of the scl.xsd.

**Comment by camillebloch** [comment]
Hi @Sander3003 

We used reference between 6-100 and 6 because 6-100 will be included in ed.3 of part 6, and to bound the 6-100 to a specific version of the part 6, containing what is expected.

We can remove this dependency but this will lead to the possibility to use 6-100 with old version of SCL which are not compliant

**Comment by Sander3003** [comment]
As discussed during the community refinement today, based on the following impressions:
- We only need a small part of 6-100 for now
- When the need to convert to the new SCL version anyway once the 6-100 is merged into the main SCL XSD.
- Java class generation was not considered during the standard developement

The decision is made to add the relevant 6-100 extensions to the CoMPAS private extensions. The documentation can mention that they are inspired by the 6-100. The 6-100 namespace is not used to prevent confusion and limit the amount of namespaces.
